src/renderer/ScreenController.tsx

```diff
@@ -241,6 +241,7 @@ export default class ScreenController extends Component {
               selectAccount: this.selectAccount,
               onAddAccount: async () => {
                 const accountId = await BackendRemote.rpc.addAccount()
+                updateDeviceChats(accountId, true) // skip changelog
                 await this.selectAccount(accountId)
```

Comment:
More a sidenote with ideas on refactoring: This stuff is crucial "business logic" (even though I hate that word) which is buried deep down in the UI between some components, maybe we can separate these things one day and have the general "onboarding" logic disconnected from the view

Reply:
sure, but such a thing needs to be done carefully with much testing.